test/functional/rpc_psbt.py

```diff
@@ -269,6 +269,10 @@ def run_test(self):
 
         self.test_utxo_conversion()
 
+        # Test that psbts with p2pkh outputs are created properly
+        p2pkh = self.nodes[0].getnewaddress(address_type='legacy')
+        psbt = self.nodes[1].walletcreatefundedpsbt([], [{p2pkh : 1}], 0, {"includeWatching" : True}, True)
+        self.nodes[0].decodepsbt(psbt['psbt'])
```

Comment:
I tried to backport, but the test passes even without the code changes.

Mind taking a look to backport this?